service/history/historyEngine.go

```diff
@@ -1158,16 +1158,15 @@ func (e *historyEngineImpl) QueryWorkflow(
 ) (retResp *types.HistoryQueryWorkflowResponse, retErr error) {
 
 	scope := e.metricsClient.Scope(metrics.HistoryQueryWorkflowScope).Tagged(metrics.DomainTag(request.GetRequest().GetDomain()))
-	shardMetricScope := e.metricsClient.Scope(metrics.HistoryQueryWorkflowScope).Tagged(metrics.ShardIDTag(strconv.Itoa(e.shard.GetShardID())))
+	shardMetricScope := e.metricsClient.Scope(metrics.HistoryQueryWorkflowScope, metrics.ShardIDTag(strconv.Itoa(e.shard.GetShardID())))
 
 	consistentQueryEnabled := e.config.EnableConsistentQuery() && e.config.EnableConsistentQueryByDomain(request.GetRequest().GetDomain())
 	if request.GetRequest().GetQueryConsistencyLevel() == types.QueryConsistencyLevelStrong {
 		if !consistentQueryEnabled {
 			return nil, workflow.ErrConsistentQueryNotEnabled
 		}
-		scope.IncCounter(metrics.ConsistentQueryPerShard)
```

Comment:
should this have been removed?  kinda seems like this is to intentionally dual-emit for more efficient total metrics.

Reply:
Yeah this should be removed. I was thinking of making it consistent with something else but it's not needed for this particular operation since it's technically not a persistence operation.